New website proposal

gmoura opened the pull request:
Hi guys! 

This is the new website proposal, with a cool and modern design, thanks @memecast to choosing this theme ❤️ . You can see the [demo here](https://hungry-borg-b29086.netlify.com/).

# Tools *&* new features

* We use [Hexo](https://hexo.io/)  to build the new website
* We can manage our site like a Blog
* We can manage our page in a easy way, because the pages is written with markdown syntax
* Easy way to deploy
* Easy way to customize the theme, because is written in [Node](https://nodejs.org/en/) and [EJS](http://ejs.co/)

Thanks for help @memecast, @mikenieva  @arun1595 and @merunga

lupomontero commented on source/_posts/CSS-Style-Guide.md:
```diff
@@ -0,0 +1,277 @@
+---
```
I'm assuming this is a _copy/paste_ from https://github.com/airbnb/css, is it? If so, I'd rather only link to the original source instead of bringing it in as a _post_ in order to prevent our "copy" from becoming _stale_ and also avoid duplicating 3rd party content. Alternatively, if we feel we need to have our own rules on top of Airbnb's guide, I'd suggest to _fork_ the upstream repo and _maintain_ our own fork as a repo.

Thoughts?

gmoura replied:
👍

lupomontero commented on source/etiquette/index.md:
```diff
@@ -0,0 +1,248 @@
+---
```
Same as CSS and JS style guides... I prefer _links_ to sources instead of duplicating content that is expected to evolve over time...

gmoura replied:
👍

lupomontero commented on source/_posts/Github-workflow.md:
```diff
@@ -0,0 +1,108 @@
+
+[learn Feature Branch Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) *by Atlassian*
+
+## Good commit messages
```
WRT to commit conventions... we have so far tried to follow/adopt the popular Angular Commit Conventions: https://gist.github.com/stephenparish/9941e89d80e2bc58a153

It might be a good idea to consider a blend between the above mentioned conventions and the ones outlined in this PR. Thoughts?

gmoura replied:
I liked this convention 👍 I will keep the discussion on #13

lupomontero commented on source/contributing/index.md:
```diff
@@ -0,0 +1,42 @@
+---
```
If this _contributing guide_ applies only to this repo, I would suggest to use the existing convention of having a `CONTRIBUTING.md` file in the repo's root, as this is _recognised_ by GitHub (as well as other collaboration tools).

gmoura replied:
👍

gmoura commented on .gitignore:
```diff
@@ -1,21 +1,3 @@
-yarn-debug.log*
-yarn-error.log*
+db.json
+public
```
Nice! I like it

gmoura commented on .gitignore:
```diff
@@ -1,21 +1,3 @@
-.env.test.local
-.env.production.local
-
-npm-debug.log*
```
Sorry! 😅 We want to exclude 😊

gmoura commented on .gitignore:
```diff
@@ -1,21 +1,3 @@
-/build
-
-# misc
-.DS_Store
```
Sorry! 😅 We want to exclude 😊

gmoura commented on _config.yml:
```diff
@@ -0,0 +1,85 @@
+## Source: https://github.com/hexojs/hexo/
+
+# Site
+title: Laboratoria Developer Chapter
```
Nice 👍

gmoura commented on _config.yml:
```diff
@@ -0,0 +1,85 @@
+
+# Site
+title: Laboratoria Developer Chapter
+subtitle: DevChapter
```
👍

gmoura commented on package.json:
```diff
@@ -1,16 +1,20 @@
 {
-  "name": "developers",
-  "version": "0.1.0",
+  "name": "hexo-site",
```
Sorry, I forgot to changed it 😅

gmoura commented on package.json:
```diff
@@ -1,16 +1,20 @@
-    "react-dom": "^16.1.0",
-    "react-scripts": "1.0.17"
+  "hexo": {
+    "version": "3.7.1"
```
I didn't notice that, but yes, it's little bit weird :sweat_smile:, but I found why this happened. When I started the "blog", I started with this command `hexo init`, after I created the first post hexo automatically created this key `"version": "3.7.1"`. 

Actually `hexo` is in the [`3.7.1` version ](https://github.com/hexojs/hexo/releases), I changed the dependencies, to get the last one.

gmoura commented on public/index.html:
```diff
@@ -1,40 +1,263 @@
 <!DOCTYPE html>
```
:scream:  I'm a bit confused to haha I will fix and I will include `public`  dir on source control

gmoura commented on source/_data/projects.json:
```diff
@@ -0,0 +1,17 @@
+[
+    {
```
I will add `.editorconfig`

gmoura commented:
@lupomontero thanks you so much for your review <3

About `package-lock.json` and `yarn`, sorry about `package-lock` I removed it and reinstall the deps with `yarn`.

I read and fixed all the things that you commented, thank you :muscle:

About the keep simple, I will think about a version with only scroll :sunglasses:

lupomontero commented:
Many thanks for the super effort @gmoura! I am going to close this issue for the time being as this "new website thing" will need to be re-assessed "after" we complete the _content_ in the `README.md` file in #26
